src/Raven.Server/Documents/Indexes/Index.cs

```diff
@@ -3350,6 +3355,22 @@ internal static bool WillResultBeAcceptable(bool isStale, IndexQueryBase<Blittab
             }
         }
 
+        protected int GetReferencesLength(HashSet<string> collections, HashSet<string> referencedCollections)
```

Comment:
It's not about the length of references but it's about the size of etags, right? So maybe let's rename to `GetReferenceEtagsSize` or something like that

Reply:
done

Reply:
done